app/assets/styles/styles.css

```diff
@@ -11,6 +11,42 @@ body {
   box-sizing: border-box;
 }
 
+.primary-nav {
+	/* to centralize the div. we have to verify what's happening. */
+	margin: auto;
+
+	overflow: hidden;
+	text-align: center;
+}
+
+/* For reduce the height of the nav */
+.logo-btn img {
```

Comment:
It is also best not to over qualify css. Instead of targeting an element by its html tag, put the class on the element you want to target. 

- `<img>` - `logo-img`
- `h1` -  `logo-text`
- `ul`  - `logo-description`
- `li` - `logo-description-text`

Reply:
Looks like you are using BEM. I would follow that pattern for all your classes.